<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesComponentEditor</class>
 <widget class="QWidget" name="PropertiesComponentEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelHeaderName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeaderIcon">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>5</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHeader">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxEnabled">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>5</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widgetComponentProperties" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>28</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolButton" name="toolButtonHeaderOptions">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>4</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../rainbow-editor.qrc">
     <normaloff>:/res/images/menu.png</normaloff>:/res/images/menu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonHeaderExpander">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>2</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../rainbow-editor.qrc">
     <normaloff>:/res/images/arrow-down.png</normaloff>:/res/images/arrow-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../rainbow-editor.qrc">
     <normaloff>:/res/images/reset.png</normaloff>:/res/images/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionDetach">
   <property name="icon">
    <iconset resource="../rainbow-editor.qrc">
     <normaloff>:/res/images/delete.png</normaloff>:/res/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Detach</string>
   </property>
   <property name="toolTip">
    <string>Detach</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="../rainbow-editor.qrc">
     <normaloff>:/res/images/help.png</normaloff>:/res/images/help.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <zorder>pushButtonHeader</zorder>
  <zorder>labelHeaderName</zorder>
  <zorder>labelHeaderIcon</zorder>
  <zorder>checkBoxEnabled</zorder>
  <zorder>widgetComponentProperties</zorder>
  <zorder>toolButtonHeaderOptions</zorder>
  <zorder>toolButtonHeaderExpander</zorder>
 </widget>
 <resources>
  <include location="../rainbow-editor.qrc"/>
 </resources>
 <connections/>
</ui>
